ONSdigital/eq-author-app
# Ear 1537 contents of other box being duplicated twice

### Context of PR ###

Fix to the duplication bug caused by having a qCode for the Checkbox Other Label

### How to review ###

1. Create a questionnaire with Radio and Checkbox Other Options.
2. Check that now, in the qCode table, there's no option to add a qCode for Checkbox Other Labels.
3. Check the publisher JSON, and now in `Detail_Answer` in the schema. Only Radio will have a qCode and checkbox will have no qCode.

What to do after everything is green
    *   Bring this branch up-to-date with Origin/Master
    *   If there are a lot of commits or some are not helpful to read, squash them down
    *   Click the Merge button on this pull request
    *   Does this change mean the Capability examples questionnaire should be updated?
    *   Move the Jira ticket for this task into the next stage of the process
